src/app/context/AuthContext.tsx

```diff
@@ -27,6 +27,24 @@ interface AuthContextType {
 const AuthContext = createContext({} as AuthContextType);
 
 export function AuthProvider({ children }: { children: React.ReactNode }) {
+  //theme
+  useEffect(() => {
+    api.getSettings().then((response) => {
```

Comment:
More like a general question: Does Alby prefer `.then()` or `async/await` or just doesn't matter?

Reply:
`.then()` was already being used for this function previously so that's why I used it, but it looks like both then and `await` are used in the codebase. Not sure which one is preferred though.

Reply:
Yeah, just noticed this. Might be good to think about a preferred way.  
@dylancom what do you think?

Definitely not blocking this.

Reply:
I think in general async/await is preferred in terms or readability and to avoid callback hell.
It's currently a bit of a mix of both in the codebase.